lib/node-package-manager.ts

```diff
@@ -53,6 +54,12 @@ export class NodePackageManager implements INodePackageManager {
 	public view(packageName: string, propertyName: string): IFuture<any> {
 		return this.loadAndExecute("view", [[packageName, propertyName], [false]]);
 	}
+
+	public executeNpmCommand(npmCommandName: string, currentWorkingDirectory: string): IFuture<void> {
+		return (() => {
+			 this.$childProcess.exec(npmCommandName, { cwd: currentWorkingDirectory }).wait();
+		}).future<void>()();
```

Comment:
You don't need to wrap this to future. Just `return this.$childProcess.exec(npmCommandName, { cwd: currentWorkingDirectory })`